config/envs/default.py

```diff
@@ -504,3 +504,5 @@ class DefaultConfig(object):
     # You can find the proper permission names from this document
     # https://docs.microsoft.com/en-us/graph/permissions-reference
     MS_GRAPH_PERMISSIONS_SCOPE = ["User.ReadBasic.All"]
+
+    GOV_NOTIFY_API_KEY = getenv("GOV_NOTIFY_API_KEY")
```

Comment:
Do we want the app to start without this (in production and/ or locally)?

I think currently the magic link step is skipped if running locally but I'm not sure if it's skipped + doesn't add anything to the queue or if it sends off the email SQS message anyway (in which case it wouldn't get through without a valid key)

I haven't thought this through at all and happy to chat about it but I don't think having a valid Notify key should be a requirement for running locally (you just wouldn't get emails) -- I think that's an accidental property of having it distributed through a queue but it would be good to explicitly make the decision either way somewhere in these changes

Reply:
I do generally think we should hard-fail if our essential configuration is missing (see long-lived post-award ticket https://mhclgdigital.atlassian.net/browse/FPASF-163 👀) I think I'd rather do that change wholesale?

I probably agree that you shouldn't need a valid Notify key locally - which is what that config flag about not sending emails is for 🫣 There's also something around whether we should have a running mock notify service ala localstack. I wish Notify supplied something like that tbh, but maybe they're just find with people using test notify keys?

Reply:
Yeah great point there is an existing flag to do that. We can always revisit these threads when we're closer to the target code layout we want to arrive at.

Nice we could almost copy that post-award ticket wholesale